frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -24,7 +24,7 @@ import { SAMPLE_COLUMN_FILTERS, SAMPLE_NEXT_STEP_FILTER_KEYS, SampleColumnID } f
 const { Text } = Typography
 
 interface LabworkStepPageProps {
-	protocol: Protocol
+	protocol: Protocol | undefined
```

Comment:
just a suggestion, maybe create a type that is for 'Protocol | undefined', and change the name since it's no longer always associated to a protocol. Maybe we can just merge and talk about it with Chris. Maybe we can have checks that always check step.type first instead of having an optional for protocol, that might change how to front-end is going to be setup.